src/libraries/System.Private.CoreLib/src/System/SpanHelpers.cs

```diff
@@ -1,7 +1,12 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if TARGET_AMD64 || TARGET_ARM64 || (TARGET_32BIT && !TARGET_ARM) || TARGET_LOONGARCH64
```

Comment:
It does not look great to have this split over multiple files. We may want to move everything together. (It can be separate change.)

Reply:
Moved to Buffer.cs

Reply:
With the subsequent changes `SpanHelpers.ClearWithoutReferences` can go back to `SpanHelpers.cs`.